hallux/proposals/simple_proposal.py

```diff
@@ -56,7 +57,13 @@ def _set_issue_lines(self):
         self.issue_lines: list[str] = copy.deepcopy(self.all_lines[self.start_line - 1 : self.end_line])
         self.proposed_lines: list[str] = copy.deepcopy(self.issue_lines)
         if self.issue.line_comment is not None:
-            self.issue_lines[self.issue.issue_line - self.start_line] += self.issue.line_comment
+            # now because self.issue_lines contain \n or \r\n at the end of each line,
+            # adding line comment gets a little bit more involved
+            issue_line = self.issue_lines[self.issue.issue_line - self.start_line]
+            issue_line = issue_line.split("\n")
```

Comment:
this dirty hack won't work on windows files.  But I don't know if we interested in those